team-builders/src/components/FormInput.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Form, FormGroup, Input, Label, Button } from 'reactstrap';
+
+const FormInput = props => {
+    console.log("props", props);
+
+    const [member, setMember] = useState({name: "", email: "", role: ""});
+
+    const inputChanges = event => {
+        console.log("event", event.target.value);
+        console.log("name", event.target.name);
+        setMember({...member, [event.target.name]: event.target.value});
+    }
+
+    console.log("member", member);
+
+    const submitMember = event => {
+        console.log("submitted");
+        event.preventDefault();
+        props.addNewMember(member);
+        setMember({ name: "", email: "", role: "" });
+    }
+
+    return (
+        <Form onSubmit={submitMember}>
+            <FormGroup>
+                <Label htmlFor="name">Name:</Label>
+                <Input type="text" placeholder="Enter name" id="name" name="name" value={member.name} onChange={inputChanges} />
+            </FormGroup>
+            <FormGroup>
+                <Label htmlFor="email">Email:</Label>
+                <Input type="email" placeholder="Enter email" id="email" name="email" value={member.email} onChange={inputChanges} />
```

Comment:
You'll notice that users can't submit an email that doesn't have an "@" symbol. Pop quiz: What in your code is causing that restriction?